### PR TITLE
Update kubervirt-user-guide to match kubervirt-kubevirt.github.io

### PR DESCRIPTION
**Pull request by jcanocan**
The current Dockerfile fails to build the kubevirt/user-guide project. This is a copy of kubevirt-kubevirt.github.io Dockerfile that allows to build the documentation using the current user-guide makefile.

Signed-off-by: Javier Cano Cano <jcanocan@redhat.com>

### CODE REVIEW

**Review by brianmcarey — COMMENTED** [review]
/lgtm

/ok-to-test

Thanks @jcanocan !

File: images/kubevirt-user-guide/Dockerfile [hunk 1]
```diff
@@ -1,33 +1,32 @@
 # BASE
-FROM registry.fedoraproject.org/fedora:34
+FROM registry.fedoraproject.org/fedora:36
 
+# COPY app configs
+WORKDIR /src
 ENV LC_ALL=en_US.UTF-8
+COPY ./_config/src/ ./
```
Comment (brianmcarey) [inline images/kubevirt-user-guide/Dockerfile:7]: The presubmit test build looks to be failing here:
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/2333/build-kubevirt-user-guide-image/1571809333564411904#1:build-log.txt%3A60

Comment (jcanocan) [reply images/kubevirt-user-guide/Dockerfile]: Not sure why this is happening. If someone knows, please give us some insights. I just updated the Dockerfile using the previous method for getting the Gemfile (`mkdir /src && cd /src && curll https://raw.../Gemfile`).

Comment (brianmcarey) [reply images/kubevirt-user-guide/Dockerfile]: I think this was seen with the kubevirt.github.io image - the presubmit and postsubmits may have to updated similiar to this  - https://github.com/kubevirt/project-infra/pull/2266

Comment (jcanocan) [reply images/kubevirt-user-guide/Dockerfile]: Thanks for the clarification. Could copying this [line](https://github.com/cwilkers/project-infra/blob/16116f56d73d004964d96df3464d0c25013088fe/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml#L195) behind this [line](https://github.com/cwilkers/project-infra/blob/16116f56d73d004964d96df3464d0c25013088fe/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml#L222) fix the issue? Same for [postsubmits](https://github.com/cwilkers/project-infra/blob/16116f56d73d004964d96df3464d0c25013088fe/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml) in the proper lines.

Comment (jcanocan) [reply images/kubevirt-user-guide/Dockerfile]: I can try in a separate PR. However, I'm not sure how to test it locally :S
Nevertheless, with the last modification I've posted yesterday the test seems to work now.

Comment (jcanocan) [reply images/kubevirt-user-guide/Dockerfile]: I've removed that line and restored `mkdir -p /src && cd /src && \
  curl https://raw.githubusercontent.com/kubevirt/user-guide/main/_config/src/Gemfile -o Gemfile && \`.

**Review by brianmcarey — APPROVED** [review]
/approve

thanks @jcanocan !

**Pull request merged**